api/diagnostics.go

```diff
@@ -786,10 +790,18 @@ func loadExternalProviders(cfg *config.Config) (externalProviders LogProviders,
 	if err = json.Unmarshal(endpointsConfig, &externalProviders); err != nil {
 		return externalProviders, err
 	}
+
+	// for debugging purpose
+	logrus.Infof("HTTPEndpoints.count: %d", len(externalProviders.HTTPEndpoints))
+	for _, endpoint := range externalProviders.HTTPEndpoints {
+		logrus.Infof("Endpoint: Port %d , endpoint url: %s FileName:%s", endpoint.Port, endpoint.URI, endpoint.FileName)
+	}
```

Comment:
remove this block

Reply:
@mnaboka 

Here is how I tested this dcos-diagnostics on window

https://github.com/soccerGB/Docs/blob/master/dcos/diagnosticsservice/HowToRunDiagnosticsOnWindows.md

Reply:
I will remove this debug only block